code/game/objects/effects/decals/cleanable/humans.dm

```diff
@@ -56,6 +56,15 @@
 
 	var/already_rotting = FALSE
 
+/obj/effect/decal/cleanable/blood/no_rot_gibs
```

Comment:
Might be better to make this a subtype of gibs with a flag set or something, not sure about that.

Reply:
I agree that it should be a subtype of gibs if possible but this is the extent of my knowledge. If I'm reading it correctly though, making it a subtype of gibs would make the proc/start_rotting trigger, but I assume that's where the flag comes in.